src/Migrations/Version20201106161146.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20201106161146 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE dataset_submission ADD remotely_hosted_url TEXT DEFAULT NULL');
+        $this->addSql("UPDATE dataset_submission SET remotely_hosted_url = dataset_file_uri WHERE dataset_file_uri LIKE 'http://%' OR dataset_file_uri LIKE 'https://%'");
```

Comment:
```suggestion
        $this->addSql("UPDATE dataset_submission SET remotely_hosted_url = dataset_file_uri WHERE dataset_file_uri SIMILAR TO 'https?:\/\/\S*'");
```
Can also use `SIMILAR TO`  to use a regular expression.